packages/gatsby-plugin-sass/README.md

```diff
@@ -94,7 +94,7 @@ plugins: [
     resolve: `gatsby-plugin-sass`,
     options: {
       postCssPlugins: [somePostCssPlugin()],
-      precision: 8,
+      precision: 8, // Set to 6 for Bootstrap 4: https://github.com/twbs/bootstrap/blob/master/site/content/docs/4.3/getting-started/theming.md#sass
```

Comment:
Could you move this to a separate example below with a note? The inline comment here could be easily missed

Reply:
Thanks, I reworked the section as per your suggestion. Let me know how it works for you now :)